revoke/revoke.go

```diff
@@ -287,15 +449,15 @@ func sendOCSPRequest(server string, req []byte, issuer *x509.Certificate) (*ocsp
 
 	switch {
 	case bytes.Equal(body, ocsp.UnauthorizedErrorResponse):
-		return nil, errors.New("OSCP unauthorized")
+		return nil, fmt.Errorf("OSCP unauthorized")
```

Comment:
`errors.New` doesn't support formatting. `fmt.Errorf` - does. Why do we need to import both `fmt` and `errors` packages?